pylab/accounts/forms.py

```diff
@@ -13,11 +13,15 @@ class UserProfileForm(forms.ModelForm):
 
     class Meta:
         model = accounts_models.UserProfile
-        fields = ('first_name', 'last_name', 'email', 'language')
+        fields = ('first_name', 'last_name', 'email', 'language', 'accepted_terms')
         help_texts = {
             'email': _(
                 "Will be used for communication. If you want not to get any emails, leave this field empty."
             ),
+            'accepted_terms': _(
+                "[Terms of serivce](http://pylab.lt/terms/). Basically, you accept that all your "
+                "creation and work done for Python workshops will belong to Python workshops."
```

Comment:
I think it is better to refer to Wikipedia TOS and additionally tosdr.org badge could be added.

Also, intellectual property can't be given to anyone and no content should belong to Python workshops, TOS should tell, that you agree that all created content will be released to public under a CC licence. And for code, terms off LICENSE file in each repository applies.

Anyway this is tricky part.